lib/ueberauth/strategy/google.ex

```diff
@@ -42,9 +42,16 @@ defmodule Ueberauth.Strategy.Google do
     end
   end
 
+  @doc """
+  Handles the callback for Google client side flow.
+  """
+  def handle_callback!(%Plug.Conn{params: %{"token" => token}} = conn) do
```

Comment:
@hassox could we add the parameter as `binary` as well? We technically don't need the `Conn` so if I add the logic of the oauth callbacks outside of an phoenix project I dont need to carry on the `Conn` everywhere.

The same for other strategies

Reply:
@yordis I'm not sure I follow when you say `binary`.  Are you asking that this match be changed from `%Plug.Conn{params: %{"token" => token}}` to `%{params: %{"token" => token}}`?

Reply:
@blakedietz could you make this change based on @yordis's comment below?

Reply:
@rohanpujaris the param that holds the JWT is called `credential` I think: https://developers.google.com/identity/gsi/web/reference/js-reference#credential